java-dataproc/google-cloud-dataproc/pom.xml

```diff
@@ -49,9 +49,14 @@
       <groupId>com.google.guava</groupId>
       <artifactId>guava</artifactId>
     </dependency>
+    <dependency>
+      <groupId>com.google.api.grpc</groupId>
+      <artifactId>proto-google-iam-v1</artifactId>
+    </dependency>
     <dependency>
       <groupId>com.google.api.grpc</groupId>
       <artifactId>grpc-google-iam-v1</artifactId>
+	  <scope>test</scope>
```

Comment:
extra indentation?

Reply:
Thanks, my editor would not show this problem. I changed this to spaces.